.github/workflows/pull-request.yaml

```diff
@@ -8,10 +8,10 @@ jobs:
   validate_metrics_docs:
     runs-on: ubuntu-latest
     steps:
-      - name: Install Golang 1.20
+      - name: Install Golang 1.21
         uses: actions/setup-go@v5
         with:
-          go-version: '1.20'
+          go-version: '1.21'
```

Comment:
You could make it use the golang version set in `go.mod`, so we don't have to touch this file in the future.

Reply:
https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file